examples/opf/experiments/anomaly/temporal/simple/description.py

```diff
@@ -195,8 +195,6 @@
             'synPermActiveInc': 0.1,
 
             'synPermInactiveDec': 0.01,
-
-            'randomSP': 1
```

Comment:
:+1:  was discussed in some other PR, this translates mostly to `learn=False in SP.compute()`